charts/quine-enterprise/values.yaml

```diff
@@ -105,6 +105,11 @@ metrics:
     # Set "host" and "port" to the reachable address of the influx server
     host: ""
     port: 8086
+  prometheus:
+    # Set to true to expose JMX metrics over HTTP on the `metricsReporterPort` using the Prometheus JMX agent
+    enabled: false
+    # Port opened up on Quine Enterprise pod, exposing Prometheus Java Agent metrics
+    metricsReporterPort: 9090
```

Comment:
Nit: How would you feel about shortening this field? The name feels a bit redundant being `metrics.prometheus.metricsReporterPort`. How about `reporterPort` or just `port`?

Reply:
`port` is a lot better. I made that change.